packages/lab/src/pagination/Paginator.tsx

```diff
@@ -54,39 +59,31 @@ export const Paginator = forwardRef<HTMLDivElement, PaginatorProps>(
     }, [page, onPageChange, count]);
 
     const isOnFirstPage = page === 1;
+
     const isOnLastPage = page === count;
 
     return (
       <div className={clsx(withBaseName(), className)} {...restProps} ref={ref}>
         {showPreviousNext && (
```

Comment:
What's the use case for pagination without previous and next buttons ?

Reply:
That's a very good point. I've set up a call with design to discuss. Thanks!

Reply:
prop removed ;)